gufe/tests/test_custom_json.py

```diff
@@ -122,7 +128,31 @@ def setup(self):
             {
                 ":is_custom:": True,
                 "__class__": "Path",
-                "__module__": 'pathlib',
-                'path': "foo/bar"
+                "__module__": "pathlib",
+                "path": "foo/bar",
+            }
+        ]
+
+
+class TestSettingsCodec(CustomJSONCodingTest):
+    pass
+
+
+class TestOpenFFQuanityCodec(CustomJSONCodingTest):
+    pass
+
+
+class TestOpenFFUnitCodec(CustomJSONCodingTest):
+    def setup(self):
+        self.codec = OPENFF_UNIT_CODEC
+        self.objs = [
+            unit.amu,
+        ]
+        self.dcts = [
+            {
+                ":is_custom:": True,
+                "__class__": "Unit",
+                "__module__": "pint.util",
+                "unit": "unified_atomic_mass_unit",
```

Comment:
@dwhswenson so something that I think is an issue that might need to be fixed in openff-units is how this class comes in, since it is showing up as pint and not an openff-units class